### PR TITLE
Bugfix/omega constraint as effect

### PR DESCRIPTION
**Pull request by adolgert**
Converts child omega constraint into a random effect by changing its mean value to the log(child mean / parent mean) for all children.

### CODE REVIEW

File: src/cascade/executor/epiviz_runner.py [hunk 1]
```diff
@@ -91,15 +92,28 @@ def add_mortality_data(model_context, execution_context, sex_id):
...
     """
-    MATHLOG.debug(f"Add omega constraint from age-standardized death rate data.")
-    MATHLOG.debug("Assigning standard error from measured upper and lower.")
+    MATHLOG.debug(f"Add omega constraint from age-standardized death rate data. "
+                  f"Assigning standard error from measured upper and lower.")
+    # Call this the long way so it can be monkeypatched in testing.
     asdr = meas_bounds_to_stdev(
-        age_groups_to_ranges(execution_context, get_age_standardized_death_rate_data(execution_context))
+        age_groups_to_ranges(execution_context,
+                             cascade.input_data.db.mortality.get_age_standardized_death_rate_data(execution_context))
```
Comment (alec-deason) [inline src/cascade/executor/epiviz_runner.py]: Why fully qualified?

File: src/cascade/executor/epiviz_runner.py [hunk 2]
```diff
@@ -116,9 +130,17 @@ def add_omega_constraint(model_context, execution_context, sex_id):
 
     children = get_descendents(execution_context, children_only=True)  # noqa: F841
     children_asdr = asdr.query("node_id in @children")
+    # Transform the children to be the random effect for the rate.
+    parent_value = parent_asdr[["age_lower", "time_lower", "mean"]].rename({"mean": "parent_mean"}, axis=1)
+    parent_and_child = children_asdr.merge(parent_value, how="left", on=["age_lower", "time_lower"])
+    # This could result in an Inf value, but that's a legal value, so...
+    children_effects = parent_and_child.assign(
```
Comment (alec-deason) [inline src/cascade/executor/epiviz_runner.py]: Oh, nice. This is less complicated than I'd feared.

File: src/cascade/executor/epiviz_runner.py (hunk 1 above)
Comment (adolgert) [reply src/cascade/executor/epiviz_runner.py]: The pytest mocking fixture changes the getattr on the import. It doesn't work if the function is imported directly. Annoying. I put a comment in to that effect.

File: src/cascade/executor/epiviz_runner.py (hunk 2 above)
Comment (adolgert) [reply src/cascade/executor/epiviz_runner.py]: It gets simple when parent and effects are on the same grid, and we defined them, which puts them on the same grid.

**Pull request merged**